package.json

```diff
@@ -61,6 +61,11 @@
     "svelte-loader": "3.1.2"
   },
   "dependencies": {
+    "@opentelemetry/api": "^1.0.4",
+    "@opentelemetry/context-zone": "^1.0.1",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.27.0",
+    "@opentelemetry/instrumentation-document-load": "^0.27.1",
+    "@opentelemetry/sdk-trace-web": "^1.0.1",
```

Comment:
I don't see these being used in this pull request. Are these relevant?

Reply:
sorry I wasn't supposed to have committed those tests with opentelemetry. I'm reverting this file entirely